### DISCUSSION TITLE
[QUESTION] Regarding the Samsung SPC-6000A emulated machine

### DISCUSSION THREAD

**Discussion by ppgrainbow** [86Box/86Box#1265] [Q&A]
I have a question to ask regarding the Samsung SPC-6000A emulated machine.

It's a early machine from 1990 that uses the CS8230 chipset like the ECS 386/32 and the maximum amount of RAM on the CS8230 chipset is **16 MB**. The S800 AT386 Modular BIOS can also only test up to 16,000 KB (15.63 MB) if more than 16 MB of RAM is installed.

Personally, I don't think that any of the early 80386-based machines had 32 MB or more RAM until at least mid-1991 or 1992.

Since the SPC-6000A uses the C&T CS8320 chipset, can you reduce the maximum amount of RAM from 32 MB to 16 MB?

Thank you for your time.

**Comment by ghost** [comment]
The specifications of most machines have to be studied.

**Reply by ppgrainbow** [reply]
I agree. I'm gonna have to privately note Greatpsycho on the PCem forums to provide information regarding the correct specifications no the Samsung SPC-6000A citing the RAM limitations of the CS8230 chipset that it uses.

If I get a response from Greatpsycho, I'll let you know.

**Reply by ppgrainbow** [reply]
Greatpsycho over at the PCem forums just notified me that although he doesn't have the actual SPC-6000A machine, the motherboard can handle up to 8 memory slots using 4 MB SIMM memory models for a maximum total of 32 MB.

If more than 16 MB of RAM is installed on the motherboard, the Power On Self-Test procedure than only test up to 16 MB.

How odd. :|
